src/qt/frame.cpp

```diff
@@ -31,6 +31,7 @@ class wxQtMainWindow : public wxQtEventSignalHandler< QMainWindow, wxFrame >
     wxQtMainWindow( wxWindow *parent, wxFrame *handler );
 
 private:
+    virtual bool focusNextPrevChild(bool) { return false; }
```

Comment:
The function name here should probably start with a capital letter. But OTOH is it really used? I could only find it mentioned in a comment.